anyio.c

```diff
@@ -229,7 +229,7 @@ board_t *anyio_get_dev(board_access_t *access, int board_number) {
     return NULL;
 }
 
-int anyio_dev_write_flash(board_t *board, char *bitfile_name, int fallback_flag, int fix_boot_flag) {
+int anyio_dev_write_flash(board_t *board, char *bitfile_name, int fallback_flag, int fix_boot_flag, int sha256_check_flag) {
```

Comment:
A future refinement would be to convert _all_ flags to c99 `bool` type. however, this style of using `int` for a flag exists before this proposed change, and you followed the style, so that's just fine as well.